### ISSUE TITLE
screw size for CB1/CM4

### ISSUE THREAD

**Issue by jscancella** [bigtreetech/Manta-M8P#28]
What screw size should be used for the CB1/CM4 mount? I tried 2mm and 2.5mm but neither seemed to want to screw in.

**Comment by jfneptune** [comment]
The boards just snap in place.  It takes a little more force than I care for, but it will just snap in place.  No screws are needed.

**Comment by jscancella** [comment]
> The boards just snap in place. It takes a little more force than I care for, but it will just snap in place. No screws are needed.

Yes I did that, but I saw that there are 4 mounting holes for screws and I wanted to use those as well.

**Comment by thisiskeithb** [comment]
M2.5, but there is tape covering the standoffs that will need to be removed.

**Comment by anscarlett** [comment]
Strange that it isn't written on the dimensional drawings anywhere. Disappointing that the screws weren't supplied with the board.

**Comment by jpereira-ca** [comment]
> M2.5, but there is tape covering the standoffs that will need to be removed.

Yes, there are tiny little round translucent orange stickers protecting the screw holes. After removing them, the M2.5 screws fit perfectly.

**Comment by anscarlett** [comment]
For reference, I found a review on amazon saying that with the heatsink, you need 2x M2.5x8mm and 2x m2.5x6mm screws. Would have been nice to have designed the heatsink so only one size screw was needed.
